src/tools/contentProcessing/GltfUtilities.ts

```diff
@@ -563,4 +635,122 @@ export class GltfUtilities {
       decodeMatrices
     );
   }
+
+  static replaceLegacyGaussianSplattingExtensionGlb(glbBuffer: Buffer): Buffer {
```

Comment:
I'll create a PR on top of this one, with comments for these. 

I know: Some of the comments that I have inserted elsewhere (and the ones that I'll insert here) are "shallow". I won't deny that I'm sometimes ""documenting"" some function like `doSomeComplexOperation` with `/** Does some complex operation */`. These comments are "open-ended", in some way (listing preconditions and postconditions, and providing context). But ... I'll do it. 

(In my local (Java) development, I've configured Eclipse to emit a warning when _anything_ is undocumented, and still maintain these "0 warnings". If someone calls me pedantic and/or claims that this is useless, we can arrange a call with a screen-sharing session, to go through some code (e.g. CesiumJS))